src/server/rpc/rpc-manager.js

```diff
@@ -68,13 +68,9 @@ RPCManager.prototype.loadRPCs = function() {
     return fs.readdirSync(PROCEDURES_DIR)
         .map(name => [name, path.join(PROCEDURES_DIR, name, name+'.js')])
         .filter(pair => fs.existsSync(pair[1]))
-        .map(pair => {
-            let service = require(pair[1]);
-            service._docs = new Docs(pair[1]);
-            return [pair[0], service];
-        })
+        .map(pair => [pair[0], pair[1], require(pair[1])])  // name, path, module
         .filter(pair => {
-            let [name, service] = pair;
+            let [name, /*path*/, service] = pair;
             if (typeof service === 'function' || !!service && !_.isEmpty(service)) {
```

Comment:
is this where the service was being thrown out?

Reply:
Yeah, if it was empty (`{}`) then this entire if statement would be skipped and it would just return false for the filtering function.

Reply:
However, if there were docs for it, the object was never empty and it would always be returned (even without any actual RPCS).